Source/Cake.AddinDiscoverer/exclusionlist.json

```
@@ -49,7 +49,8 @@
     "Cake.Xrm.DataMigration",
     "Cake.Xrm.Sdk",
     "Cake.Xrm.SolutionPackager",
-    "Cake.Xrm.Spkl"
+    "Cake.Xrm.Spkl",
+    "Cake.Watch_Advance"
```

Can you please add a comment that it is excluded because it is a fork of Cake.Watch:

```suggestion
    "Cake.Watch_Advance" // This is a clone of Cake.Watch
```

---

Sure